src/script/sign.h

```diff
@@ -302,6 +302,9 @@ struct PSBTInput
                     } else if (key.size() != 1) {
                         throw std::ios_base::failure("Non-witness utxo key is more than one byte type");
                     }
+                    // Set the stream to unserialize with witness since this is always a valid network transaction
```

Comment:
You can instead use `UnserializeFromVector(OverrideStream(&s, s.GetType(), s.GetVersion() & ~SERIALIZE_TRANSACTION_NO_WITNESS);` to avoid permanently changing the flags on `s` (similar below).

Reply:
Done